app/gateway/http/favorites.go

```diff
@@ -15,20 +15,20 @@ func ListFavorite(ctx *gin.Context) {
 	var req pb.FavoriteListReq
 	if err := ctx.Bind(&req); err != nil {
 		log.LogrusObj.Errorf("Bind:%v", err)
-		ctx.JSON(http.StatusOK, ctl.RespError(ctx, err, "绑定参数错误"))
+		ctx.JSON(http.StatusInternalServerError, ctl.RespError(ctx, err, "绑定参数错误"))
```

Comment:
don't change this `http.StatusOK` to `http.StatusInternalServerError` , we all set the HTTP response code is 200, and  we will set the logic of response code in response body, you can see the `ctl.RespError()`